### PR TITLE
fix: first spike of jx 3.x docs

### PR DESCRIPTION
**Pull request by jstrachan**

### CODE REVIEW

File: content/en/docs/v3/guides/benefits.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+* We can use the `helm list` command line to view versions of each chart/app nicely in the CLI.
+  * we can avoid composite charts to simplfiy configuration and upgrades
+* Everything is now an app. So if you want to remove our `nginx-ingress` chart and replace it with another ingress solution (knative / istio / gloo / ambassador / linkerd or whatever) just go ahead and use the [apps commands](/docs/v3/guides/apps/) to add/remove apps and have boot manage everything in a consistent way
+    * e.g. here's [an example](https://github.com/jstrachan/environment-bucketrepo-dev/blob/master/helmfile.yaml#L2-L5) of removing `nginx`, `chartmusem` and `nexus` and adding in `istio`, `flagger` and some other tools like `bucketrepo` and `kuberhealthy` via a single simple yaml change. Incidentally to replace the use of `Ingress` resources with istio's `Gateway` and `VirtualService` you also need to add the `kind: istio` flag in the [jx-requirements.yml](https://github.com/jstrachan/environment-bucketrepo-dev/blob/master/jx-requirements.yml#L57) file
```
Comment (ankitm123) [inline content/en/docs/v3/guides/benefits.md:15]: Will this repo move to the jx (or similar) org?

Comment (jstrachan) [reply content/en/docs/v3/guides/benefits.md]: ah - let me switch to the jenkins x equivalent - great catch!

File: content/en/docs/v3/guides/jenkins/_index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: Jenkins
+linktitle: Jenkins
+type: docs
+description: Improved Jenkins and Jenkins X interoperability
+weight: 100
+---
+
+
+The [jx 3.x binary](/docs/v3/guides/jx3/) includes the initial version of the [Trigger Jenkins Proposal](https://github.com/jstrachan/enhancements/blob/jenkins-trigger/proposals/trigger-jenkins/README.md) to improve the interoperability of Jenkins and Jenkins X
```
Comment (ankitm123) [inline content/en/docs/v3/guides/jenkins/_index.md:10]: I think this also needs to be changed, right? (github org/username)

File: content/en/docs/v3/guides/jenkins/overview.md [hunk 3]
```diff
@@ -4,17 +4,7 @@ linktitle: Overview
...
-**This code should not be used in production, or be adopted for usage.  It should only be used to provide feedback to the Labs team.**
-
-Thank you for your participation,
-
--Labs
-
-
-{{% /alert %}}
 
 The [Trigger Jenkins Proposal](https://github.com/jstrachan/enhancements/blob/jenkins-trigger/proposals/trigger-jenkins/README.md) aims to improve the interoperability of Jenkins and Jenkins X so you can use both together to solve all of your CI/CD needs.
```
Comment (ankitm123) [inline content/en/docs/v3/guides/jenkins/overview.md:9]: Same .. github name ..

**Pull request merged**